### PR TITLE
Fix #960: Remove code-level consistency check in Innovatrics Document Provider

### LINKED ISSUE #960: Remove code-level consistency check in Innovatrics Document Provider
For example Czech Identity Card has ČESKÁ REPUBLIKA in the Nationality field in visual zone, but CZE in machine readable zone. That creates false alarm in text consistency check of Innovatrics document verification. To overcome this issue https://github.com/wultra/enrollment-server/issues/907 implemented postprocessing of the text consistency check. However this should be managed on Innovatrics instance configuration level. The example bellow excludes `documentNumber` filed from consistency check of SVK identity card 2022 edition.

```yml
innovatrics:
  dot:
    dis:
      customer:
        document:
          inspection:
            text-consistency-check:
              SVK_identity-card_2022-12-01:
                exclusions:
                  - documentNumber
```

### PR DESCRIPTION
**Pull request by jnpsk**
Fix #960 by removing code setting a set of field to which the text consistency check is relevant.

### CODE REVIEW

File: docs/onboarding/Configuration-Properties.md [hunk 1]
```diff
@@ -144,7 +144,6 @@ The Onboarding Server uses the following public configuration properties:
...
 | `enrollment-server-onboarding.provider.innovatrics.documentVerificationConfiguration.documentCountries` | `CZE`                                                                                                     | List of expected countries of issue of identification documents as three-letter country codes, i.e. ISO 3166-1 alpha-3. If empty, all countries of issue known to Innovatrics are considered during classification, which may have negative impact on performance. |
-| `enrollment-server-onboarding.provider.innovatrics.document-verification-configuration.crucialFields`   | `documentNumber`, `dateOfIssue`, `dateOfExpiry`, `surname`, `dateOfBirth`, `personalNumber`, `givenNames` | Set of fields in camelCase that are cross-validated between the machine-readable zone and visual zone. Only those specified fields that are actually extracted from a document are considered. If empty, no inconsistencies will be checked.                       |
```
Comment (banterCZ) [inline docs/onboarding/Configuration-Properties.md:147]: I am wondering where and how to document  Innovatrics instance configuration. I do not mean to copy&paste their documentation, but highlight keywords and record our expectations.

**Review by banterCZ — COMMENTED** [review]
The change looks good, but let's think about Innovatrics documentation a little bit.

File: docs/onboarding/Configuration-Properties.md (hunk 1 above)
Comment (jnpsk) [reply docs/onboarding/Configuration-Properties.md:147]: Created dedicated issue https://github.com/wultra/enrollment-server/issues/964

**Review by banterCZ — APPROVED** [review]
Ok, Innovatrics documentation will be done in a separate issues.

**Pull request merged**